explainer.md

```diff
@@ -268,18 +270,35 @@ A big codebase tend to evolve slowly and soon becomes legacy code. Old code vs n
 
 Modifying code to resolve a conflict (e.g.: global variables) is non-trivial, specially in big codebases.
 
-The Realms API can provide a _lightweight_ mechanism to preserve the integrity of the intrinsics.0 Therefore, it could isolate libraries or logical pieces of the codebase per Realm.
+The Realms API can provide a _lightweight_ mechanism to preserve the integrity of the intrinsics. Therefore, it could isolate libraries or logical pieces of the codebase per Realm.
 
 ### <a name='Templatelibraries'></a>Template libraries
```

Comment:
I still think that this example should be removed. It will likely initially be too expensive to instantiate this many Realms; we don't want to be telling developers that this is expected to be cheap.

Reply:
I can remove, but it doesn't require many Realms. You could freeze a realm to operate the templates inside.